media-video/mpv/mpv-9999.ebuild

```diff
@@ -138,11 +138,11 @@ PATCHES=(
 
 mpv_check_compiler() {
 	if [[ ${MERGE_TYPE} != "binary" ]]; then
-		if tc-is-gcc && ( [[ $(gcc-major-version) -lt 4 ]] || \
-			( [[ $(gcc-major-version) -eq 4 ]] && [[ $(gcc-minor-version) -lt 5 ]] ) ); then
+		if tc-is-gcc && { [[ $(gcc-major-version) -lt 4 ]] || \
+			{ [[ $(gcc-major-version) -eq 4 ]] && [[ $(gcc-minor-version) -lt 5 ]]; }; }; then
```

Comment:
Minor nit but you do know that you can put `&&` and `||` (and plain parentheses `( ... )`) inside `[[ ... ]]`, right?